programs/token/src/instructions/set_authority.rs

```diff
@@ -52,9 +52,10 @@ impl<'a> SetAuthority<'a> {
         ];
 
         // instruction data
-        // -  [0]: instruction discriminator
-        // -  [1]: authority_type
-        // -  [2..35] new_authority
+        // -  [0]: instruction discriminator (1 byte, u8)
+        // -  [1]: authority_type (1 byte, u8)
+        // -  [2]: new_authority presence flag (1 byte, AuthorityType)
+        // -  [3..35] new_authority (optional 32 bytes, Pubkey)
```

Comment:
Same here, a `,` after `optional`? So it would look like `(optional, 32 bytes, Pubkey)`.